tycho-targetplatform/src/main/java/org/eclipse/tycho/targetplatform/TargetDefinitionFile.java

```diff
@@ -447,6 +452,16 @@ public boolean includeAllEnvironments() {
         public boolean includeSource() {
             return includeSource;
         }
+
+        @Override
+        public boolean includeConfigurePhase() {
+            return includeConfigurePhase;
+        }
+
+        @Override
+        public Boolean followRepositoryReferences() {
```

Comment:
Maybe an enum would be better with value

- DEFAULT
- ENABLED
- DISABLED

Boolean objects tend to be causing NPE if devs are not carefull enough.

Reply:
Good idea, done